README.md

```diff
@@ -339,7 +339,11 @@ To test the senders/receivers using Kafka, you might either use the docker image
 sbt kafka/docker:publishLocal
 ```
 
-Then, go to `docker/kafka` and run `docker-compose up`. This will start zookeeper, the kafka broker,
+Then, go to `docker/kafka` and run
+```
+docker-compose -f ../metrics/docker-compose.metrics.yml -f docker-compose.yml -p kafka up
```

Comment:
**question**: Do we need that `-p kafka`?

Reply:
otherwise the name of the whole container will be taken from the first docker-compose file - so it will be named prometheus